source/tests/system/nirfmxlte_driver_api_tests.cpp

```diff
@@ -85,6 +102,1192 @@ TEST_F(NiRFmxLTEDriverApiTests, InitializeFromNIRFSA_Close_Succeeds)
+  EXPECT_SUCCESS(session, client::cfg_reference_level(stub(), session, "", 0.0));
+  EXPECT_SUCCESS(session, client::cfg_external_attenuation(stub(), session, "", 0.0));
+  EXPECT_SUCCESS(session, client::cfg_digital_edge_trigger(stub(), session, "", DIGITAL_EDGE_TRIGGER_SOURCE_PFI0, DIGITAL_EDGE_TRIGGER_EDGE_RISING, 0.0, false));
+  EXPECT_SUCCESS(session, client::cfg_number_of_subblocks(stub(), session, "", NUMBER_OF_SUBBLOCKS));
+  for (int i = 0; i < NUMBER_OF_SUBBLOCKS; i++) {
+    auto subblock_string_response = client::build_subblock_string(stub(), "", i);
+    EXPECT_SUCCESS(session, subblock_string_response);
+    EXPECT_SUCCESS(session, client::cfg_frequency(stub(), session, subblock_string_response.selector_string_out(), subblocks_center_frequency[i]));
+    EXPECT_SUCCESS(session, client::cfg_subblock_frequency_definition(stub(), session, subblock_string_response.selector_string_out(), subblocks_frequency_definition[i]));
+    EXPECT_SUCCESS(session, client::cfg_component_carrier_spacing(stub(), session, subblock_string_response.selector_string_out(), COMPONENT_CARRIER_SPACING_TYPE_NOMINAL, -1));
+    EXPECT_SUCCESS(session, client::cfg_number_of_component_carriers(stub(), session, subblock_string_response.selector_string_out(), NUMBER_OF_COMPONENT_CARRIERS));
+    EXPECT_SUCCESS(session, client::cfg_component_carrier_array(stub(), session, subblock_string_response.selector_string_out(), {20e+6}, {0.0}, std::vector<int32>()));
+    EXPECT_SUCCESS(session, client::sem_cfg_number_of_offsets(stub(), session, subblock_string_response.selector_string_out(), NUMBER_OF_OFFSET_SEGMENT));
+    EXPECT_SUCCESS(session, client::sem_cfg_offset_frequency_array(stub(), session, subblock_string_response.selector_string_out(), {15e+3, 1.5e+6, 5.5e+6, 20.5e+6}, {985e+3, 4.5e+6, 19.5e+6, 24.5e+6}, subblocks_sideband));
+    EXPECT_SUCCESS(session, client::sem_cfg_offset_rbw_filter_array(stub(), session, subblock_string_response.selector_string_out(), {10e+3, 250e+3, 250e+3, 250e+3}, subblocks_rbw_filter_type));
+    EXPECT_SUCCESS(session, client::sem_cfg_offset_bandwidth_integral_array(stub(), session, subblock_string_response.selector_string_out(), {3, 4, 4, 4}));
+    EXPECT_SUCCESS(session, client::sem_cfg_offset_absolute_limit_array(stub(), session, subblock_string_response.selector_string_out(), {-19.5, -8.5, -11.5, -23.5}, {-19.5, -8.5, -11.5, -23.5}));
+  }
+  EXPECT_SUCCESS(session, client::cfg_link_direction(stub(), session, "", LINK_DIRECTION_UPLINK));
+  EXPECT_SUCCESS(session, client::select_measurements(stub(), session, "", MEASUREMENT_TYPES_SEM, true));
+  EXPECT_SUCCESS(session, client::sem_cfg_sweep_time(stub(), session, "", SEM_SWEEP_TIME_AUTO_TRUE, 1e-3));
+  EXPECT_SUCCESS(session, client::sem_cfg_averaging(stub(), session, "", SEM_AVERAGING_ENABLED_FALSE, 10, SEM_AVERAGING_TYPE_RMS));
+  if (linkDirection == NIRFMXLTE_INT32_LINK_DIRECTION_UPLINK)
```

Comment:
Same here, delete `linkDirection` and keep body of `if` or `else`. Preferably the opposite of what you choose for SEMContiguousMultiCarrierFromExample_FetchData_DataLooksReasonable.